data/tools/python/config/producer_test.yml

```diff
@@ -0,0 +1,13 @@
+# Test configuration for the frame producer
+---
+  log_level: debug
+  log_format: "%(asctime)s.%(msecs)03d %(levelname)-5s %(name)s - %(message)s"
+  shared_buffer: frame_producer
+  shared_mem_size: 16000000
+  shared_buffer_size: 4000000
+  boost_mmap_mode: false
+  ready_endpoint: tcp://127.0.0.1:5001
+  release_endpoint: tcp://127.0.0.1:5002
+  frames: 1000
+  testfile_path: '/aeg_sw/work/projects/inaira/odin-inaira/data/Test-Images/'
```

Comment:
This exposes internal details of our systems - the path should be generic for committed files